scripts/e2e-test.sh

```diff
@@ -16,8 +16,8 @@ REPORTSDIR=$(realpath "$SCRIPTDIR/..")
 #TESTS="install basic_volume_io csi replica rebuild node_disconnect/replica_pod_remove uninstall"
 ALL_TESTS="install basic_volume_io csi resource_check replica rebuild uninstall"
```

Comment:
should add io_soak to ALL_TESTS. Or maybe rename ALL_TESTS to DEFAULT_TESTS

Reply:
...or both

Reply:
I vacillating on that, it would mean that if you do not specify any tests (most developers would probably do that I think) then you end up with a 30 minute + test run, unless one modifies the config
Should soak tests be part of the default?